app/2.0/start/quick-tour.md

```diff
@@ -211,7 +211,7 @@ The template repeater (`dom-repeat`) is a specialized template that binds to an
 </demo-tabs>
 
 * Change the first and last names inside this.employees
-* Add another employee by inserting this text into the array definition after Tony Morelli:<br/>
+* Add another employee by inserting the following text into the array definition after Tony Morelli:<br/>
```

Comment:
Didn't notice this in the sample before, but please make sure the sample names aren't real people... https://developers.google.com/style/examples